invenio_rdm_records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -123,19 +123,152 @@
 
       }
     },
-    "_created": {
+    "dates": {
+      "description": "Date interval.",
+      "items": {
+        "additionalProperties": false,
+        "properties": {
+          "description": {
+            "description": "Description of the date interval.",
+            "type": "string"
+          },
+          "end": {
+            "description": "End date.",
+            "type": "string",
+            "format": "date-time"
+          },
+          "start": {
+            "description": "Start date.",
+            "type": "string",
+            "format": "date-time"
+          },
+          "type": {
+            "description": "Type of the date interval."
+          }
+        },
+        "required": [
+          "type"
+        ],
+        "type": "object"
+      },
+      "type": "array"
+    },
+    "description": {
+      "description": "Description/abstract for record.",
```

Comment:
Librarians on our side favour just "Description for record." here since "abstract" will go in the `"additional_descriptions"` section.

Reply:
Do they always have description + abstract? I think there is the chance that some use the abstract as description, no?

Reply:
Talked IRL, removing "abstract"

Reply:
Talked IRL, removing "abstract"